grr/server/grr_response_server/gui/ui/components/flow_args_form/osquery_query_helper/table_info_item/table_info_item.ng.html

```diff
@@ -0,0 +1,26 @@
+<div class="item-header">
```

Comment:
I think it's ok to put table_info_item.* files into osquery_query_helper/ folder without creating an additional subfolder. At the moment we don't have a rule of "one folder for one component".